include/ck/utility/amd_gemm_dpp.hpp

```diff
@@ -11,33 +11,57 @@ namespace ck {
 
 namespace dpp8 {
 
-template <index_t MPerLanegroup,
-          index_t NPerLanegroup,
-          index_t KPerLanegroup,
-          class FloatA,
-          class FloatB,
-          class FloatVecC,
+template <class ABDataType>
+struct dpp_datatypes;
+
+template <>
+struct dpp_datatypes<half_t>
+{
+    // Dot product of `half2_t` and `half2_t` to get `float`. Reducing 2 elements from K in a
+    // single instruction.
+    using a_dtype                        = half_t;
+    using b_dtype                        = half_t;
+    using c_dtype                        = float;
+    static constexpr index_t k_per_instr = 2;
+};
+
+template <index_t MPerThread,
+          index_t NPerThread,
+          index_t KPerThread,
+          class BaseInputType,
+          class AVecDataType,
+          class BVecDataType,
+          class CVecDataType,
           bool ShareA>
-__device__ void RunGemm(const FloatA& a, const FloatB& b, FloatVecC& c_vec)
+struct DppInstrRunner
```

Comment:
Maybe it's more like `DppWavewiseGemm` or even `DppLanegroupGemm` ?

Reply:
Renamed to `DppLanegroupGemm `